packages/audioplayers/tizen/src/audio_player.cc

```diff
@@ -167,29 +166,25 @@ void AudioPlayer::SetDataSource(std::vector<uint8_t> &data) {
 }
 
 void AudioPlayer::SetVolume(double volume) {
-  if (volume_ != volume) {
```

Comment:
You removed the comparison between old and new values in general. could you please explain the reason in detail?

Reply:
The old value (cached value) becomes invalid when audio is replayed in loop mode because "playback rate" (and possibly volume, I haven't checked) is automatically reset to 1.0. If playback rate was set to 2.0 during previous iteration, calling `SetVolume(2.0)` at next iteration will not change the playback rate properly because of the cached value.